furusiya/config.json

```diff
@@ -22,7 +23,8 @@
         "lightRadius": 10,
         "statsPointsOnLevelUp": 5,
         "startingArrows": 10,
-        "stallionFollowDistance": 3
+        "stallionFollowDistance": 3,
+        "restingRatio": 1 // formula: this * max_hp / current_hp

```

Comment:
I would rather just `+= 1` and have a value like `healthGainedPerRest`, but it should be a percentage ... I guess that's what you have. Changing the name to `percent` here would communicate that better.